docs/understanding-the-magic.rst

```diff
@@ -52,6 +52,16 @@ This allows you to access the session within the context of an incoming
 HTTP request, but it will *not* allow you access it outside that
 context.
 
+State & Security
+================
+
+One of the key features of :attr:`~OAuth2ConsumerBlueprint.session` is that 
+the requests it generates use a `state` variable to ensure that the source
```

Comment:
In reStructuredText, you use double-backticks to make something code-formatted. It should look like this:

    use a ``state`` variable